src/scenes/HomePage/components/Works/components/Item/Item.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Img from 'gatsby-image';
+import PropTypes from 'prop-types';
+
+import styles from './Item.module.scss';
+
+const Item = ({ images }) => {
+  return (
+    <div className={`${styles.item} swiper-slide`}>
+      {images.map((image, index) => {
+        return (
+          <Img
+            data-name={index}
+            key={image.childImageSharp.fluid + index}
```

Comment:
weird key again https://tppr.me/8JSPx. I think this should work fine
```suggestion
            key={image.childImageSharp.fluid.src}
```